app/assets/javascripts/application.js

```diff
@@ -20,4 +20,10 @@
 //= require nprogress
 //= require nprogress-turbolinks
 //= require main
-//= require jquery.sidr.min
\ No newline at end of file
+//= require jquery.sidr.min
+
+$(document).ajaxError(function (e, xhr, settings) {
```

Comment:
I'm okay if you want to leave this as it is - if and when a use case does pop up, we can change it :)